Formula/gwenhywfar.rb

```diff
@@ -17,24 +18,32 @@ class Gwenhywfar < Formula
     sha256 mojave:        "1439d26ec6d62bfe1da89457eb782c2ef1b4b1a2d6a9de99dee9ae3c5bce290c"
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
   depends_on "gettext"
   depends_on "gnutls"
   depends_on "libgcrypt"
   depends_on "openssl@1.1"
   depends_on "pkg-config" # gwenhywfar-config needs pkg-config for execution
+  depends_on "qt"
+
+  patch do
```

Comment:
What is this patch for, and when can it be removed? Please document this in a comment.